lib/tasks/vote.rake

```diff
@@ -0,0 +1,25 @@
+require 'ffaker'
+namespace :vote do
+  desc 'Vote for specific debate and answer'
+
+  task :positive, [:debate_id, :auth_token] => [:environment] do |_, args|
+    debate = Debate.find(args[:debate_id])
+    auth_token = args[:auth_token] ? args[:auth_token] : debate.auth_tokens.create!
+    v = VoteService.new(answer: debate.answers.positive.first, auth_token: auth_token)
```

Comment:
VoteService expects `AuthToken` instance so when You pass string from arguments it won't work